src/fontra/views/editor/editor.js

```diff
@@ -2436,8 +2456,9 @@ export class EditorController {
     const numGuidelines = guidelineSelection?.length || 0;
     // + (fontGuidelineSelection?.length || 0);
 
-    const s = numGuidelines > 1 ? "s" : "";
-    return `${hasLockedGuidelines ? "Unlock" : "Lock"} Guideline${s}`;
+    return hasLockedGuidelines
```

Comment:
Same, pls put the switch inside the `translatePlural()` call.

Reply:
I missed a couple notes from you, because they were hidden. Now, this is done as well. Sorry.